### PR TITLE
lightdm: 1.24.0 -> 1.26.0

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
Needed an update.

I most likely didn't do this correctly.
Comments and corrections are very welcome.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/display-managers/lightdm/default.nix [hunk 1]
```diff
@@ -1,33 +1,64 @@
...
+    vala
+  ];
+  
   buildInputs = [
-    pam libxcb glib libXdmcp itstool libxml2 libxklavier libgcrypt
-    qt4 libaudit
+    glib
+    gnome3.yelp_tools
+    gnome3.yelp_xsl
+    itstool
```
Comment (jtojnar) [inline pkgs/applications/display-managers/lightdm/default.nix]: gnome3.yelp_tools gnome3.yelp_xsl itstool libtool which and possibly libxml2 should go to `nativeBuildInputs`

**Review by jtojnar — APPROVED** [review]

File: pkgs/applications/display-managers/lightdm/default.nix (hunk 1 above)
Comment (worldofpeace) [reply pkgs/applications/display-managers/lightdm/default.nix]: i removed libxml2 now. I don't think it needed it.

**Comment by worldofpeace** [comment]
I guess it's optional to also build with accountsservice. Should I make that an option?

**Comment by jtojnar** [comment]
I would just add it non-optionally.

**Comment by worldofpeace** [comment]
Ok, I think I did all the things.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -18249,7 +18249,7 @@ with pkgs;
   insync = callPackage ../applications/networking/insync { };
 
   lightdm = libsForQt5.callPackage ../applications/display-managers/lightdm {
-    qt4 = null;
+    qt4 = false;
```
[the remark this answers is not in the document]
Comment (worldofpeace) [reply pkgs/top-level/all-packages.nix]: done

**Comment by jtojnar** [comment]
@GrahamcOfBorg test lightdm

**Pull request merged**